### PR TITLE
fix: homepage navigation  #741

### LINKED ISSUE #741: 🐛 [Bug report]: Glitch in homepage navigation bar
### Descrcibe the bug?

In the homepage, while clicking on the heart icon in the navigation bar, the navigation bar moves right.

### Steps to reproduce the bug?

1. Go to `http://localhost:3000` or `https://reactplay.io/` on laptop/desktop browser.
2. Click on `heart icon` on the top right.
3. You will find that navigation bar is moving to the right.

### Expected behavior

Navigation bar shouldn't change position because of clicking actions.

### Desktop (Please provide your system information)

OS: MacOS
Browser: Chrome
Version: 106.0.5249.119

### Mobile (Please provide your device information)

_No response_

### Screenshot / Screenshare

_No response_

### Relevant log output

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by dev0869**
Glitch came during click heart shape button fixed. I made Changes in Header Folder

Fixes # #741

https://github.com/reactplay/react-play/assets/108985813/63a7f051-14e8-4e07-82a9-07622024ed62

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -70,6 +70,7 @@
     "redux-persist": "^6.0.0",
     "remarkable": "^2.0.1",
     "reselect": "^4.1.8",
+    "sass": "^1.69.6",
```
Comment (priyankarpal) [inline package.json:73]: Can you please tell me what that is?

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (dev0869) [reply package.json:73]: > Can you please tell me what that is?

Yes, when I install dependencies then I got an error about node-sass.Then someone says me that node-sass is deprecated that make me unable to install dependencies of node modules.then to fix that I install sass and then I able to install dependencies that's it.

Comment (dev0869) [reply package.json:73]: I just installed it to run my application because my node modules not installed makes node-sass error.

Comment (priyankarpal) [reply package.json:73]: ok

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Review by Sachin-chaurasiya — APPROVED** [review]

**Pull request merged**